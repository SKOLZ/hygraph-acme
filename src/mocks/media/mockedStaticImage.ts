export const mockedStaticImage = {
  __typename: "StaticImage" as const,
  image: {
    url: "https://media.graphassets.com/bbd3PKNcTBGVkNbcois7",
    width: 480,
    height: 320,
  },
};
