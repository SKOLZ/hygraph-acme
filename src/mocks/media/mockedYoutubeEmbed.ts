export const mockedYoutubeEmbed = {
  __typename: "YoutubeEmbed" as const,
  title: "Media Title",
  youtubeUrl: "https://www.youtube.com/embed/znkZ8X_7tLA",
};
