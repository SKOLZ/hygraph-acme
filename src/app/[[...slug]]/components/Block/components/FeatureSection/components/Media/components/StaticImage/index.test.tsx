import { mockedStaticImage } from "@/mocks/media/mockedStaticImage";
import { StaticImage } from ".";
import { render } from "@testing-library/react";

describe("StaticImage", () => {
  it("renders the element correctly", () => {
    const {container} = render(<StaticImage {...mockedStaticImage} />);
    expect(container).toMatchSnapshot();
  })
})
