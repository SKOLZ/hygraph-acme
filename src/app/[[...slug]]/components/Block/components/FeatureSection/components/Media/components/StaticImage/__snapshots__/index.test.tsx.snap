// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StaticImage renders the element correctly 1`] = `
<div>
  <img
    alt=""
    class="rounded border border-gray-50 min-w-0 lg:m-10"
    data-nimg="1"
    decoding="async"
    height="320"
    loading="lazy"
    src="/_next/image?url=https%3A%2F%2Fmedia.graphassets.com%2Fbbd3PKNcTBGVkNbcois7&w=1080&q=75"
    srcset="/_next/image?url=https%3A%2F%2Fmedia.graphassets.com%2Fbbd3PKNcTBGVkNbcois7&w=640&q=75 1x, /_next/image?url=https%3A%2F%2Fmedia.graphassets.com%2Fbbd3PKNcTBGVkNbcois7&w=1080&q=75 2x"
    style="color: transparent;"
    width="480"
  />
</div>
`;
